dataland-frontend/src/components/forms/parts/kpiSelection/EuTaxonomyKPIsModel.ts

```diff
@@ -188,18 +188,18 @@ export const euTaxonomyKpiInfoMappings = {
   alignedRevenue:
     "Percentage of the revenue that is taxonomy-aligned, i.e., generated by an eligible economic activity that is making a substantial contribution to at least one of the climate and environmental objectives, while also doing no significant harm to the remaining objectives and meeting minimum standards on human rights and labour standards.",
   totalRevenue:
-    "Total revenue in MM (Millions) for the financial year. i.e., income arising in the course of an entity's ordinary activities, the amounts derived from the sale of products and the provision of services after deducting sales rebates and value added tax and other taxes directly linked to turnover. Overall turnover is equivalent to a firm's total revenues over some period of time.",
+    "Total revenue for the financial year. i.e., income arising in the course of an entity's ordinary activities, the amounts derived from the sale of products and the provision of services after deducting sales rebates and value added tax and other taxes directly linked to turnover. Overall turnover is equivalent to a firm's total revenues over some period of time.",
```

Comment:
Die sind im Excel alle drei noch nicht angepasst.

Reply:
`. i.e.,` steht so im data dictionary. Ist aber schlicht falsch, oder?
Das müsste doch `. I.e.,` sein.

Reply:
Ich habe im Code alle "i.e." die nach einem Punkt kommen, zu "I.e." geändert.  Waren mehrere.

Das Excel file bearbeite ich als nächstes.

Reply:
Ich habe jetzt:

- Alle "in MM (Millions)" die hier im Pull Request entfernt wurden, auch im Excel entfernt.
- Alle `. i.e.` auch im Excel zu `. I.e.` geändert.

Passt?

Reply:
Die Einheiten Columns fehlen noch